README.md

```diff
@@ -112,15 +112,18 @@ This string will be assumed to be the `exec` property of that method.
 
 The following options can be specified over the CLI:
 
--  `-d`, `--cwd` becomes `data.start.working_directory`
--  `-e`, `--env` appends to `data.start.environment`
--  `-g`, `--group` becomes `data.start.group`
+-  `-c`, `--category` becomes `data.category`
+-  `-D`, `--dependency` appends to `data.dependencies`
+-  `-d`, `--cwd` becomes `data.working_directory`
+-  `-e`, `--env` appends to `data.environment`
+-  `-g`, `--group` becomes `data.group`
 -  `-i`, `--ident` becomes `data.ident`
 -  `-l`, `--label` becomes `data.label`
--  `-p`, `--privilege` appends to `data.start.privileges`
+-  `-p`, `--privilege` appends to `data.privileges`
 -  `-S`, `--stop` becomes `data.stop.exec`
 -  `-s`, `--start` becomes `data.start.exec`
--  `-u`, `--user` becomes `data.start.user`
+-  `-r`, `--refresh` becomes `data.refresh.exec`
```

Comment:
nit, this is no longer alphabetically sorted

Reply:
I'll ensure that all of it is sorted